azurerm/internal/services/powerbidedicated/registration.go

```diff
@@ -0,0 +1,23 @@
+package powerbidedicated
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+type Registration struct{}
+
+// Name is the name of this Service
+func (r Registration) Name() string {
+	return "PowerBIDedicated"
+}
+
+// SupportedDataSources returns the supported Data Sources supported by this Service
+func (r Registration) SupportedDataSources() map[string]*schema.Resource {
+	return map[string]*schema.Resource{}
+}
+
+// SupportedResources returns the supported Resources supported by this Service
+func (r Registration) SupportedResources() map[string]*schema.Resource {
+	return map[string]*schema.Resource{
+		"azurerm_powerbi_dedicated_capacity": resourceArmPowerBIDedicatedCapacity()}
```

Comment:
Looking there seems to be premium and embedded but not dedicated, so what should go here?
```suggestion
		"azurerm_powerbi_????_capacity": resourceArmPowerBIDedicatedCapacity()}
```

Reply:
Also is capacity the right term, again looking at the portal:
![image](https://user-images.githubusercontent.com/1638467/75816788-b262bd80-5d4a-11ea-9565-f34609b65ceb.png)

it seems like this is a `powerbi_embeded_project` regardless of what the API is calling it?

Reply:
yes. it's the powerbi embedded project. I think we should updated to "azurerm_powerbi_embedded" if we don't consider of what the API is calling. And this resource created in azure portal is called "Power BI Embedded". Does it make sense?
![image](https://user-images.githubusercontent.com/19754191/75844362-b35e2280-5e10-11ea-9fd0-25d8885c7bec.png)

Below is my understanding:
power bi -> Premium    -> workspace
               -> Embedded -> capacity